runtime/lua/vim/treesitter.lua

```diff
@@ -197,13 +197,20 @@ end
 --- Each capture is represented by a table containing the capture name as a string as
 --- well as a table of metadata (`priority`, `conceal`, ...; empty if none are defined).
 ---
----@param bufnr number Buffer number (0 for current buffer)
----@param row number Position row
----@param col number Position column
+--- If called without argument, returns the captures at cursor.
 ---
----@return table[] List of captures `{ capture = "capture name", metadata = { ... } }`
+---@param bufnr number|nil Buffer number (0 for current buffer)
+---@param row number Position row (default cursor position if `bufnr` is `nil)
+---@param col number Position column (default cursor position if `bufnr` is `nil)
```

Comment:
```suggestion
---@param row number|nil Position row (default cursor position if `bufnr` is `nil`)
---@param col number|nil Position column (default cursor position if `bufnr` is `nil`)
```

Reply:
Yeah, I was planning to rework this to be similar to how I ended up implementing the semantic tokens functions.